tools/docker/Dockerfile.integ-test

```diff
@@ -55,6 +55,7 @@ COPY firecracker-control/cmd/containerd/firecracker-ctr /usr/local/bin/ctr
 
 COPY runtime/firecracker-runc-config.json.example /etc/containerd/firecracker-runc-config.json
 COPY tools/image-builder/rootfs.img /var/lib/firecracker-containerd/runtime/default-rootfs.img
+COPY tools/docker/do_not_edit.config.json /etc/containerd/firecracker-runtime.json
```

Comment:
We shouldn't need to put a firecracker-runtime config file here, we generate them during each test case now: https://github.com/firecracker-microvm/firecracker-containerd/blob/e80d4cc83f1207c0b9cd5eb1e5bacc67e55e6415/runtime/integ_test.go#L56

Reply:
This is needed due to the `firecracker-local` package which is loaded up before the tests can write the config. I've renamed this for a more appropriate name and created a github issue, #353, so we can eventually move away from this config file.